forge-gui/res/cardsfolder/upcoming/ramses_assassin_lord.txt

```diff
@@ -0,0 +1,9 @@
+Name:Ramses, Assassin Lord
+ManaCost:2 B U
+Types:Legendary Creature Human Assassin
+PT:4/4
+K:Deathtouch
+S:Mode$ Continuous | Affected$ Assassin.Other+YouCtrl | AddPower$ 1 | AddToughness$ 1 | Description$ Other Assassin's you control get +1/+1.
+T:Mode$ LosesGame | ValidPlayer$ Player.Other+wasDealtCombatDamageThisTurnBy Creature.Assassin+YouCtrl | TriggerZones$ Battlefield | Execute$ TrigWinGame | TriggerDescription$ Whenever a player loses the game, if they were attacked this turn by an Assassin you controlled, you win the game. 
```

Comment:
attacked this turn and combat damage is not the same

Will need more engine support

Reply:
Removed it